ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -44,7 +44,7 @@
   <LinkedBlock
     @params={{array backend.backendLink backend.id}}
     class="list-item-row linkable-item is-no-underline"
-    data-test-auth-backend-link={{backend.id}}
+    data-test-secrets-backend-link={{backend.id}}
```

Comment:
This was bugging me since there is the same selector for auth mounts so I updated it.